### ISSUE TITLE
Style tag duplication for same styles

### ISSUE THREAD

**Issue by fyapy** [denn1s/vite-plugin-linaria#2]
### Current Behavior

When we change file with Linaria styles, we will get new style tag on each change.
For example, our changes will be like this:
`width: 543px;` -> `width: 545px;`(new style tag) -> `width: 546px;`(new style tag) -> `width: 545px;`(nothing happens)

![image](https://user-images.githubusercontent.com/30187840/117363844-d0372f80-aec5-11eb-8dcf-c478b64225fd.png)

### Desired Behavior
Update existed style tag, and update style tag in `width: 545px; (nothing happens)` case.

**Comment by fyapy** [comment]
Sorry, but i don't know how to link my pull request to fix issue😅
PR: https://github.com/denn1s/vite-plugin-linaria/pull/3

**Comment by denn1s** [comment]
I am unable to replicate the duplicating issue, have tried on several different setups, would you happen so have a sample project?

after testing the pull request I noticed that handling the css in that way makes it so that it does not go through the rest of the rollup plugin chain, therefore, some features such as postcss stop working. I want to see an example of the duplication to see if I can figure what is causing it in the first place

**Comment by fyapy** [comment]
Of course, here is the repository, and a video with a playback example

Example repo: https://github.com/fyapy/vite-plugin-linaria-reproduction
Video: https://drive.google.com/file/d/1M3HlVY-y7AUEDW6uZyROFxTomxLW3UvD/view

**Comment by fyapy** [comment]
As I understand it, vite assigns an id to each style. But our id is calculated by creating a hash from the style file. And thus, every time we change something in the styles, we get a new id, thus vite considers that this is a completely new css, and creates a new tag, and does not update the old one.
